src/main/scala/circt/Convention.scala

```diff
@@ -0,0 +1,52 @@
+package chisel3.circt
+
+import chisel3.RawModule
+import chisel3.experimental.{annotate, ChiselAnnotation}
+import firrtl.annotations.{ModuleName, SingleTargetAnnotation}
+import firrtl.FirrtlUserException
+
+/** Annotation to force the compiler to use a specified port convention.
+  * May only be applied to modules.
```

Comment:
can you be a little more precise on this comment? any subclass of `BaseModule`? (can I put this annotation on a BlackBox)?